Supporting Resources/v2.0/Datamodel/Datafiles/Activity I-DM-2-A.yml

```diff
@@ -20,17 +20,17 @@ title:
 
 #Describe the benefit that is achieved by implementing this activity
 benefit: 
-  You have overview about security defects affecting applications throughout the whole organization. 
+  You have overview of security defects with severity rating affecting applications throughout the entire organization and you define when to act upon them.
```

Comment:
The benefit is over 20 words like this (we were aiming for 15). I'd suggest to remove " and you define when to act upon them." as this is not a benefit (= reward) for me, rather more work.

Reply:
Done